src/components/project/form-row.vue

```diff
@@ -0,0 +1,149 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <tr class="project-form-row">
+    <!--todo: come back and figure out all the permissions-->
+    <td class="name">
+      <link-if-can :to="primaryFormPath(form)">
+        {{ form.nameOrId() }}
+      </link-if-can>
+    </td>
+    <template v-if="form.publishedAt != null">
+      <td class="review-state">
```

Comment:
```vue
<td v-for="reviewState of reviewState" :key="reviewState" class="review-state">
  <router-link :to="submissionsPath[reviewState]">
    {{ $n(form.reviewStates[reviewState], 'default') }}<span :class="reviewStateIcon(reviewState)"></span>
  </router-link>
</td>
```

You'll need to define a computed property to return `reviewStates` ([received, hasIssues, edited]).